.travis.yml

```diff
@@ -44,6 +45,12 @@ matrix:
       python: 3.7
       env: NODE_GYP_FORCE_PYTHON=python3 EXPERIMENTAL_NODE_GYP_PYTHON3=1
       before_install: nvm install 12
+    - name: "Python 3.7 on macOS"
+      os: osx
+      osx_image: xcode11
```

Comment:
I know that I am trying your patience but... can you please do me a favor comment out line 50.  The error messages (and investigations elsewhere) suggest to me that using the current xcode11 might be our problem.

https://docs.travis-ci.com/user/reference/osx/#xcode-version
```
+No receipt for 'com.apple.pkg.DeveloperToolsCLILeo' found at '/'.
    +No receipt for 'com.apple.pkg.DeveloperToolsCLI' found at '/'.
    +gyp: No Xcode or CLT version detected!
```

Reply:
No problem. Pushed a commit just to get a Travis run, I will discard it if xcode11 is not the problem.

The Python 2 version also uses xcode11, so I don't think this will fix it. If it does, we have to understand why it's not a problem with Python 2.